doc/Plotter.ipynb

```diff
@@ -123,6 +123,33 @@
      "prompt_number": 4
     },
     {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Use an additional parameter to specify the color of the line in rgb: \"event_name:column:color\".  The color is specified as a comma separated of rgb values, from 0 to 255.  E.g. 255,0,0 is red and 86,40,32 is brown.  Similar to the example above, to plot the temperature in purple and the previous temperature in green you can do:"
```

Comment:
Something like that can be added to plotter::ILinePlot.py to complete the docstring for the "signals" param.